DATA/common/setenv_calib.sh

```diff
@@ -114,6 +115,11 @@ if [[ $BEAMTYPE != "cosmic" ]] || [[ $FORCECALIBRATIONS == 1 ]] ; then
   if [[ $CAN_DO_CALIB_ZDC_TDC == 1 ]]; then
     if [[ -z ${CALIB_ZDC_TDC+x} ]]; then CALIB_ZDC_TDC=1; fi
```

Comment:
I think for disabling by default, we should set it to =0 here

Reply:
do you mean `CAN_DO_CALIB_FT0_TIMEOFFSET=0` - this var?